apps/src/util/experiments.js

```diff
@@ -34,7 +34,6 @@ experiments.I18N_TRACKING = 'frontend-i18n-tracking';
 experiments.TIME_SPENT = 'time-spent';
 experiments.BYPASS_DIALOG_POPUP = 'bypass-dialog-popup';
 experiments.SPECIAL_TOPIC = 'special-topic';
-experiments.OPT_IN_EMAIL_REG_PARTNER = 'optInEmailRegPartner';
```

Comment:
What was this for?

Reply:
[This](https://github.com/code-dot-org/code-dot-org/pull/42034) was one of the first things I worked on when I started volunteering at Code.org like 4 years ago. It was an additional checkbox in the teacher signup flow to ask users if they wanted us to share their contact info with their regional partner (we didn't end up going through with it). We originally had it behind an experiment flag (and connected to Optimizely).

Reply:
Love love loving the cleanup (and you having the context from 4 years ago to know this should also be removed). Thank you!!